2020-09-15 08:33:48,302 DEBUG - (leriza.muralla) - Successfully loaded and updated table
2020-09-15 08:36:00,421 DEBUG - (leriza.muralla) - Successfully loaded and updated table
2020-09-15 08:36:06,233 DEBUG - (leriza.muralla) - Successfully loaded and updated table
2020-09-15 08:36:16,196 DEBUG - (leriza.muralla) - Successfully loaded and updated table
2020-09-15 08:36:33,716 DEBUG - (leriza.muralla) - Successfully loaded and updated table
2020-09-15 09:29:14,130 DEBUG - starting thread (client mode): 0xd251d490L
2020-09-15 09:29:14,131 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2020-09-15 09:29:14,140 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_7.4
2020-09-15 09:29:14,140 INFO - Connected (version 2.0, client OpenSSH_7.4)
2020-09-15 09:29:14,144 DEBUG - kex algos:[u'curve25519-sha256', u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group16-sha512', u'diffie-hellman-group18-sha512', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha256', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'rsa-sha2-512', u'rsa-sha2-256', u'ssh-dss', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] server encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] client mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] server mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2020-09-15 09:29:14,144 DEBUG - Kex agreed: curve25519-sha256@libssh.org
2020-09-15 09:29:14,145 DEBUG - HostKey agreed: ssh-ed25519
2020-09-15 09:29:14,145 DEBUG - Cipher agreed: aes128-ctr
2020-09-15 09:29:14,145 DEBUG - MAC agreed: hmac-sha2-256
2020-09-15 09:29:14,145 DEBUG - Compression agreed: none
2020-09-15 09:29:14,157 DEBUG - kex engine KexCurve25519 specified hash_algo <built-in function openssl_sha256>
2020-09-15 09:29:14,158 DEBUG - Switch to new keys ...
2020-09-15 09:29:14,163 DEBUG - Adding ssh-ed25519 host key for [119.28.27.203]:28032: 1b13146a61b7ab58444e46560da54f14
2020-09-15 09:29:14,165 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2020-09-15 09:29:14,202 DEBUG - userauth is OK
2020-09-15 09:29:14,226 INFO - Authentication (publickey) successful!
2020-09-15 09:29:14,231 DEBUG - [chan 0] Max packet in: 32768 bytes
2020-09-15 09:29:14,235 DEBUG - Received global request "hostkeys-00@openssh.com"
2020-09-15 09:29:14,235 DEBUG - Rejecting "hostkeys-00@openssh.com" global request from server.
2020-09-15 09:29:14,236 DEBUG - Debug msg: Adding to environment: NAME=argus
2020-09-15 09:29:14,278 DEBUG - [chan 0] Max packet out: 32768 bytes
2020-09-15 09:29:14,278 DEBUG - Secsh channel 0 opened.
2020-09-15 09:29:14,302 DEBUG - [chan 0] Sesch channel 0 request ok
2020-09-15 09:29:14,310 DEBUG - [chan 0] Sesch channel 0 request ok
2020-09-15 09:29:14,311 DEBUG - [chan 1] Max packet in: 32768 bytes
2020-09-15 09:29:14,314 DEBUG - [chan 1] Max packet out: 32768 bytes
2020-09-15 09:29:14,314 DEBUG - Secsh channel 1 opened.
2020-09-15 09:29:14,334 DEBUG - [chan 1] Sesch channel 1 request ok
2020-09-15 09:29:14,366 DEBUG - [chan 1] EOF received (1)
2020-09-15 09:29:14,366 DEBUG - [chan 1] EOF sent (1)
2020-09-15 09:29:14,367 DEBUG - EOF in transport thread
2020-09-15 09:29:14,377 DEBUG - starting thread (client mode): 0xd2c20810L
2020-09-15 09:29:14,377 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2020-09-15 09:29:14,389 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_7.4
2020-09-15 09:29:14,389 INFO - Connected (version 2.0, client OpenSSH_7.4)
2020-09-15 09:29:14,393 DEBUG - kex algos:[u'curve25519-sha256', u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group16-sha512', u'diffie-hellman-group18-sha512', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha256', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'rsa-sha2-512', u'rsa-sha2-256', u'ssh-dss', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] server encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] client mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] server mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2020-09-15 09:29:14,393 DEBUG - Kex agreed: curve25519-sha256@libssh.org
2020-09-15 09:29:14,393 DEBUG - HostKey agreed: ssh-ed25519
2020-09-15 09:29:14,393 DEBUG - Cipher agreed: aes128-ctr
2020-09-15 09:29:14,393 DEBUG - MAC agreed: hmac-sha2-256
2020-09-15 09:29:14,394 DEBUG - Compression agreed: none
2020-09-15 09:29:14,407 DEBUG - kex engine KexCurve25519 specified hash_algo <built-in function openssl_sha256>
2020-09-15 09:29:14,408 DEBUG - Switch to new keys ...
2020-09-15 09:29:14,412 DEBUG - Adding ssh-ed25519 host key for [119.28.27.203]:28032: 1b13146a61b7ab58444e46560da54f14
2020-09-15 09:29:14,412 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2020-09-15 09:29:14,452 DEBUG - userauth is OK
2020-09-15 09:29:14,828 INFO - Authentication (publickey) successful!
2020-09-15 09:29:14,834 DEBUG - [chan 0] Max packet in: 32768 bytes
2020-09-15 09:29:14,837 DEBUG - Received global request "hostkeys-00@openssh.com"
2020-09-15 09:29:14,838 DEBUG - Rejecting "hostkeys-00@openssh.com" global request from server.
2020-09-15 09:29:14,838 DEBUG - Debug msg: Adding to environment: NAME=argus
2020-09-15 09:29:14,879 DEBUG - [chan 0] Max packet out: 32768 bytes
2020-09-15 09:29:14,880 DEBUG - Secsh channel 0 opened.
2020-09-15 09:29:14,906 DEBUG - [chan 0] Sesch channel 0 request ok
2020-09-15 09:29:14,914 DEBUG - [chan 0] Sesch channel 0 request ok
2020-09-15 09:29:14,915 DEBUG - [chan 1] Max packet in: 32768 bytes
2020-09-15 09:29:14,918 DEBUG - [chan 1] Max packet out: 32768 bytes
2020-09-15 09:29:14,918 DEBUG - Secsh channel 1 opened.
2020-09-15 09:29:14,926 DEBUG - [chan 1] Sesch channel 1 request ok
2020-09-15 09:29:14,968 DEBUG - [chan 1] EOF received (1)
2020-09-15 09:29:14,968 DEBUG - [chan 1] EOF sent (1)
2020-09-15 09:29:14,969 DEBUG - [chan 2] Max packet in: 32768 bytes
2020-09-15 09:29:14,981 DEBUG - [chan 2] Max packet out: 32768 bytes
2020-09-15 09:29:14,981 DEBUG - Secsh channel 2 opened.
2020-09-15 09:29:14,993 DEBUG - [chan 2] Sesch channel 2 request ok
2020-09-15 09:29:15,005 DEBUG - [chan 2] EOF received (2)
2020-09-15 09:29:15,005 DEBUG - [chan 2] EOF sent (2)
2020-09-15 09:29:15,006 DEBUG - [chan 3] Max packet in: 32768 bytes
2020-09-15 09:29:15,050 DEBUG - [chan 3] Max packet out: 32768 bytes
2020-09-15 09:29:15,050 DEBUG - Secsh channel 3 opened.
2020-09-15 09:29:15,080 DEBUG - [chan 3] Sesch channel 3 request ok
2020-09-15 09:29:15,092 DEBUG - [chan 3] EOF received (3)
2020-09-15 09:29:15,093 DEBUG - [chan 3] EOF sent (3)
2020-09-15 09:29:15,094 DEBUG - EOF in transport thread
2020-09-15 09:46:23,040 DEBUG - (quincy) - Successfully loaded and updated table
2020-09-15 09:53:14,789 DEBUG - starting thread (client mode): 0xd28cf350L
2020-09-15 09:53:14,790 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2020-09-15 09:53:14,799 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_7.4
2020-09-15 09:53:14,799 INFO - Connected (version 2.0, client OpenSSH_7.4)
2020-09-15 09:53:14,804 DEBUG - kex algos:[u'curve25519-sha256', u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group16-sha512', u'diffie-hellman-group18-sha512', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha256', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'rsa-sha2-512', u'rsa-sha2-256', u'ssh-dss', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] server encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] client mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] server mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2020-09-15 09:53:14,804 DEBUG - Kex agreed: curve25519-sha256@libssh.org
2020-09-15 09:53:14,804 DEBUG - HostKey agreed: ssh-ed25519
2020-09-15 09:53:14,804 DEBUG - Cipher agreed: aes128-ctr
2020-09-15 09:53:14,804 DEBUG - MAC agreed: hmac-sha2-256
2020-09-15 09:53:14,804 DEBUG - Compression agreed: none
2020-09-15 09:53:14,816 DEBUG - kex engine KexCurve25519 specified hash_algo <built-in function openssl_sha256>
2020-09-15 09:53:14,817 DEBUG - Switch to new keys ...
2020-09-15 09:53:14,823 DEBUG - Adding ssh-ed25519 host key for [119.28.27.203]:28032: 1b13146a61b7ab58444e46560da54f14
2020-09-15 09:53:14,824 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2020-09-15 09:53:14,861 DEBUG - userauth is OK
2020-09-15 09:53:14,885 INFO - Authentication (publickey) successful!
2020-09-15 09:53:14,890 DEBUG - [chan 0] Max packet in: 32768 bytes
2020-09-15 09:53:14,895 DEBUG - Received global request "hostkeys-00@openssh.com"
2020-09-15 09:53:14,895 DEBUG - Rejecting "hostkeys-00@openssh.com" global request from server.
2020-09-15 09:53:14,896 DEBUG - Debug msg: Adding to environment: NAME=argus
2020-09-15 09:53:14,937 DEBUG - [chan 0] Max packet out: 32768 bytes
2020-09-15 09:53:14,938 DEBUG - Secsh channel 0 opened.
2020-09-15 09:53:14,963 DEBUG - [chan 0] Sesch channel 0 request ok
2020-09-15 09:53:14,971 DEBUG - [chan 0] Sesch channel 0 request ok
2020-09-15 09:53:14,971 DEBUG - [chan 1] Max packet in: 32768 bytes
2020-09-15 09:53:14,975 DEBUG - [chan 1] Max packet out: 32768 bytes
2020-09-15 09:53:14,975 DEBUG - Secsh channel 1 opened.
2020-09-15 09:53:14,994 DEBUG - [chan 1] Sesch channel 1 request ok
2020-09-15 09:53:20,221 DEBUG - [chan 2] Max packet in: 32768 bytes
2020-09-15 09:53:20,223 DEBUG - [chan 1] EOF received (1)
2020-09-15 09:53:20,223 DEBUG - [chan 1] EOF sent (1)
2020-09-15 09:53:20,225 DEBUG - [chan 2] Max packet out: 32768 bytes
2020-09-15 09:53:20,225 DEBUG - Secsh channel 2 opened.
2020-09-15 09:53:20,274 DEBUG - [chan 2] Sesch channel 2 request ok
2020-09-15 09:53:20,286 DEBUG - [chan 2] EOF received (2)
2020-09-15 09:53:20,286 DEBUG - [chan 2] EOF sent (2)
2020-09-15 09:53:20,288 DEBUG - EOF in transport thread
2020-09-15 09:54:43,749 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 09:56:42,854 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 09:57:25,408 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 09:57:47,778 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 09:58:23,986 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 09:58:47,555 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 09:59:12,478 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 10:10:53,143 DEBUG - (quincy) - Successfully loaded and updated table
2020-09-15 10:13:24,349 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 11:22:23,581 DEBUG - (laird) - Successfully loaded and updated table
2020-09-15 11:51:01,524 ERROR - Internal Server Error: /esync/acc-midpay_nginx/generate/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 287, in post
    nonstandard_esync_log(hostgroup, request.user.username, [result[1]],activity_type='CREATE')
  File "/app/argus_final/app/argus/esync/lib/esync_logger.py", line 20, in nonstandard_esync_log
    f.write(i)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 202-206: ordinal not in range(128)
2020-09-15 11:51:38,346 ERROR - Internal Server Error: /esync/acc-midpay_nginx/generate/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 287, in post
    nonstandard_esync_log(hostgroup, request.user.username, [result[1]],activity_type='CREATE')
  File "/app/argus_final/app/argus/esync/lib/esync_logger.py", line 20, in nonstandard_esync_log
    f.write(i)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 204-208: ordinal not in range(128)
2020-09-15 11:57:34,307 ERROR - Internal Server Error: /esync/acc-midpay_nginx/generate/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 287, in post
    nonstandard_esync_log(hostgroup, request.user.username, [result[1]],activity_type='CREATE')
  File "/app/argus_final/app/argus/esync/lib/esync_logger.py", line 20, in nonstandard_esync_log
    f.write(i)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 325-331: ordinal not in range(128)
2020-09-15 11:58:05,695 ERROR - Internal Server Error: /esync/acc-midpay_nginx/generate/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 287, in post
    nonstandard_esync_log(hostgroup, request.user.username, [result[1]],activity_type='CREATE')
  File "/app/argus_final/app/argus/esync/lib/esync_logger.py", line 20, in nonstandard_esync_log
    f.write(i)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 343-349: ordinal not in range(128)
